README.md

```diff
@@ -55,6 +65,7 @@ To delete all data and restart from scratch, `docker-compose rm` and `docker vol
 Finally, start the server with run `lapis server` and visit http://localhost:8080 to see the page!
 
 The code cache is currently turned off, so refreshing the page will show any changes to the code immediately.
+</details>
 
 Submit changes via PR, and happy hacking!
```

Comment:
This now double with L42

Reply:
Are you certain? This is a separate, second one that starts at L46

Reply:
I was talking about L70 which is after the details block.
It's the same text in L42 after the first details block.
So while minimized, you will see them double.
Maybe pull that before the details then?

Reply:
Sure, done.